storage/google_drive/drive_client_wrapper.py

```diff
@@ -0,0 +1,135 @@
+from googleapiclient.discovery import build
+from httplib2 import Http
+from oauth2client import file, client, tools
+from googleapiclient.http import MediaFileUpload
+
+import mimetypes
+import os
+
+# If modifying these scopes, delete the file token.json.
+SCOPES = 'https://www.googleapis.com/auth/drive'
+
+drive_service = None
+drive_folder_root = None
+
+def init_client(auth_credentials_path):
+    global drive_service
+    global drive_folder_root
+    store = file.Storage('token.json')
+    creds = store.get()
+    if not creds or creds.invalid:
+        flow = client.flow_from_clientsecrets('credentials.json', SCOPES)
+        creds = tools.run_flow(flow, store)
+    drive_service = build('drive', 'v3', http=creds.authorize(Http()))
+    drive_folder_root = drive_service.files()
+
+def get_root_id():
+    return drive_folder_root.get(fileId='root').execute().get('id')
+
+def get_children(folder_id):
```

Comment:
This is still pushing a bit too much burden on the client to understand the drive api. Consider dual mode APIs?

`get_children_by_id(folder_id)`
`get_children(folder_path)`